GHzDACs/Cleanup/dac.py

```diff
@@ -71,38 +128,39 @@
 # to raw byte strings prior to being sent to the direct ethernet server.
 
 # Time for master to delay before SRAM to ensure synchronization
-MASTER_SRAM_DELAY = 2 #us
+MASTER_SRAM_DELAY = 2  # us
 
 
 class DAC(fpga.FPGA):
-    
     MAC_PREFIX = '00:01:CA:AA:00:'
     REG_PACKET_LEN = 56
     READBACK_LEN = 70
-    
+
     TIMING_PACKET_LEN = 30
-    
+
     # Master delay before SRAM to ensure synchronization
-    MASTER_SRAM_DELAY = 2 # microseconds
+    MASTER_SRAM_DELAY = 2  # microseconds
```

Comment:
I removed this one in favor of the module-level variable. The reason is that the Memory code used it, which is not part of the DAC class structure.